starter-code/lib/viking.js

```diff
@@ -19,13 +19,96 @@
 //   FootViking: FootViking
 // };
 
-function Soldier () {}
 
-function Viking () {}
 
-function Saxon () {}
+function Soldier (name) {
+  this.name = name;
+  this.health = 300;
+  this.strength = 150;
+  this.attack = function (user) {
+  return "soldier attacks!";
+    //need to work on this
+  };
 
-function War () {}
+  this.receiveDamage = function (damage) {
+    this.health = this.health - damage;
+    return "ouch";
+
+  };
+}
+
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.constructor = Viking;
+
```

Comment:
If you use `.call` on Soldier here, then you get health and strength from soldier so there's no point in passing them in as arguments since they're getting set to 300 and 150 respectively.  Just FYI